settings/settings.go

```diff
@@ -334,6 +336,21 @@ func (n Networks) IPs() (ips []string) {
 	return
 }
 
+func (n Networks) HasInterfaceAlias() bool {
```

Comment:
Since this function is public, you need to have specific tests for it. So either you make private and it's tested in code that uses it or you have tests for it. If you keep public you also likely need to promote it in the `Networks` interface definition and make sure all struct implementing this (might only be this one) have suitable implementation and tests.